src/ui/articleview.cc

```diff
@@ -2199,11 +2199,17 @@ void ArticleView::highlightFTSResults()
     return;
   }
 
+  QString accuracy = "exactly";
+
+  if ( regString.contains( QRegularExpression( "[\\x4e00-\\x9fa5]+" ) ) ) {
```

Comment:
PCRE supports testing Unicode character properties  https://www.pcre.org/original/doc/html/pcrepattern.html

Matching `\p{han}` seems more meaningful here.

I am not with my computer now, not sure if it will works in Qt.

Reply:
I do not have my environment neither, I'll test it later.

Reply:
should be supported.
can be extended to CJK.